### ISSUE TITLE
Error "The module/namespace 'Fable.Import.React' from compilation unit 'Fable.React' did not contain the namespace, module or type 'ReactElement'" after opening Fulma module

### ISSUE THREAD

**Issue by drk-mtr** [Fulma/Fulma#203]
This is likely to be something I'm doing wrong rather than a Fulma error, so apologies in advance if this is a case of "user error".

I am trying to get a slightly hacked copy of the SAFE Bookstore app working (so it's worth mentioning that the file containing the `Client.Login` module in the code snippet below is being referenced by both the Client.fsproj and Server.fsproj in order to allow Server Side Rendering). I'm using the new versions of the following packages:

- Fable.Core: `3.0.0-beta-005`
- Fable.React: `5.0.0-beta-006`
- fable-compiler: `2.2.3`
- Fulma: `2.0.0-beta-003`

When I have the following, I get no errors:

```
module Client.Login

open Fable.React
open Fable.React.Props
```

However, when adding the open statement as below, intellisense complains, and when I roll over `module` it says `The module/namespace 'Fable.Import.React' from compilation unit 'Fable.React' did not contain the namespace, module or type 'ReactElement'`

```
module Client.Login

open Fable.React
open Fable.React.Props
open Fulma
```

Thanks in advance for any help!

**Comment by MangelMaxime** [comment]
Hello @drk-mtr, because you are using `Fable.Core v3+` you also need to use all the packages in `beta`.

The current stable version of Fulma only supports `Fable.Core v2.x`.

So in your `paket.dependencies` you  also need to mark `Fulma` as `prerelease` and probably all of the Fable related packages. You can find more infos [here](https://fable.io/blog/Announcing-2-2.html)

**Comment by drk-mtr** [comment]
Thanks for taking a look! Unfortunately I still see the issue when using `Fulma 2.0.0-beta-003`. My `paket.dependencies`:

```
  nuget Fable.Core 3.0.0-beta-005
  nuget Fable.Elmish prerelease
  nuget Fable.Elmish.Browser prerelease
  nuget Fable.Elmish.React 3.0.0-beta-4
  nuget Fable.Elmish.HMR prerelease
  nuget Fable.React prerelease
  nuget Fable.Promise prerelease
  nuget Fable.Fetch prerelease
  nuget Fulma prerelease
```

I'll wipe my cache completely and have another go I guess.

Huge thanks for all the high quality free software by the way!

**Comment by MangelMaxime** [comment]
Hum, I would not pin `Fable.Core` and `Fable.Elmish.React` to a specific reason so everything could use the latest version.

If needed you can take a look at [fulma-demo](https://github.com/MangelMaxime/fulma-demo) which was ported to `Fable.Core v3+`

> Huge thanks for all the high quality free software by the way!

Thank you :) And really happy that you use them :)

I will try to update the `SAFE-Bookstore` to see if something is missing for SAFE support. Because like I said `Fulma-demo` has been ported and at work we are also using `Fable 2.2` and `Fable.Core v3`.

**Comment by MangelMaxime** [comment]
Just to be sure, you did run `paket.exe update` in order to update your deps?

**Comment by drk-mtr** [comment]
I don't think this is anything to do with Fulma, so thanks for the pointers.

I could see that when I removed `storage: none` in Paket, the `/packages` folder confirmed that I had the `Fulma 2.0.0-beta-003` package. However, when I did a `fake build -t run`, I could see it created `.fable/Fulma.1.1.0` (and the same happened if I deleted the `.fable` folder and retried from scratch). I had tried multiple `paket install --force` and `paket update` etc. so I don't know why it was hanging on to 1.1.0.

So I think it was Ionide getting upset about the fact that `Fulma.1.1.0` references `Fable.Helpers.React.Props`.

After doing a `paket clear-cache`, deleting `/.fable` folder, deleting `paket.lock`, deleting all `/obj` and `/bin` folders and starting afresh it is now creating `Fulma.2.0.0-beta-003` in the `/.fable` folder as it should.

Thanks again!
